search/search_repository.go

```diff
@@ -524,8 +553,39 @@ func (q Query) generateExpression() (criteria.Expression, error) {
 	return res, nil
 }
 
+func extractQueryOptions(q Query) *QueryOptions {
+	qo := QueryOptions{}
+	for _, o := range q.Children {
+		switch o.Name {
+		case "parent-exists":
+			qo.ParentExists = false
+			if *o.Value == "true" {
```

Comment:
Should this be string "true" or boolean "true"?

Reply:
Currently values should be string.